src/App.js

```diff
@@ -1,16 +1,44 @@
 import React from 'react';
 import './App.css';
 import chatMessages from './data/messages.json';
+import ChatLog from './components/ChatLog';
+import { useState } from 'react';
 
 const App = () => {
+  const [messageData, setMessages] = useState(chatMessages);
+
+  const updateMessages = (updatedData) => {
+    const changedMessages = messageData.map((message) => {
+      if (message.id === updatedData.id) {
+        return updatedData;
+      } else {
+        return message;
+      }
+    });
```

Comment:
I might consider refactoring this, so that it doesn't update ALL of the message data, just the `liked` property